src/views/MainPage/components/UserContents.tsx

```diff
@@ -1,91 +1,121 @@
-import { useEffect } from 'react';
+import React, { useEffect } from 'react';
 import { useInView } from 'react-intersection-observer';
 import { styled } from 'styled-components';
 
 import { APPLY_STATUS } from '../constants/applyStatus';
 import { DesignerResponse, ModelResponse } from '../hooks/type';
 
-import { OfferCard, ApplicationCard } from './Card';
+import { Card } from './Card';
 
-interface UserContentsProps {
-  data: DesignerResponse | ModelResponse;
+interface DesignerContentsProps {
+  data: DesignerResponse;
```

Comment:
MainPage에서 mainContent에 조건부를 넣지 않기 위해 여기서 undefined 유니온 처리를 할 수 있습니다. 

```suggestion
  data: DesignerResponse | undefined;
```

Reply:
사실 이건 피하고 싶은 방법이긴했지만 승희님이 적어주신 것을 보고 type safety에 대해 조금 더 찾아봤는데
undefined 상태를 적절히 처리할 수 있도록 구현한다면 괜찮은 방법인 것 같습니다!
밑에 적어주신 것처럼 if (!data) return; 을 통해 아래의 코드에서 값이 존재함을 보장해 줄 수 있으니 좋네요! 
이 방법은 처음 봤는데 코드에 많은 조건문을 추가하지 않고도 보장해 줄 수 있겠네요 감사합니다! ☺️